posts/2024-07-30-24.0.0.8-beta.adoc

```diff
@@ -0,0 +1,808 @@
+
+The next beta release of websocket-2.2 will limit the link:https://jakarta.ee/specifications/websocket/2.2/apidocs/server/jakarta/websocket/onmessage#maxMessageSize([MaxMessageSize]) to `Integer#MAX_VALUE` for the `OnMessage` annotation to avoid type errors.
+
+To enable this feature, add the following code to your `server.xml` file.
+
+[source,xml]
+----
+ <featureManager>
+    <feature>websocket-2.2</feature>
+</featureManager>
+----
+
+For more information, such as the Javadocs, specification documents, and other details,see link:https://jakarta.ee/specifications/websocket/2.2/[Jakarta WebSocket 2.2].
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/28921
+// Contact/Reviewer: Riva-Tholoor-Philip, jhanders34
+// // // // // // // //
+
+[#jpa]
+== Use Jakarta Persistance with improved type safety and other enhancements in Jakarta Persistance 3.2
+
+The Jakarta Persistence API is a richly featured API which provides object-model approach to persisting, fetching, and modifying data stored on a relational database system.
+
+Jakarta Persistence 3.2 (`persistence-3.2`) includes the following updates:
+
+* Improve type safety of the find() and refresh() methods
+* Provide programmatic way to provide Schema Management
+* Allow Java records as Embeddable Types
+* Improvements of JPQL/Criteria API - add additional functions and operations, such as `left()`, `right`, `replace`, `||` `operator`, `cast`, `union`, `union all`, `intersect`, `intersect all`, `except`, and  `except all`
+* Support for subqueries in the `select`, `from`,  and `join` clauses.
+
+To use the `persistence-3.2` feature, add the following code to your `server.xml` file:
+
+[source,xml]
+----
+<featureManager>
+   <feature>persistence-3.2</feature>
+</featureManager>
+----
+
+This feature uses Eclipselink 5.0 and the JPA provider. If you wish to bring your own 3rd party persistence provider, enable the `persistenceContainer-3.2` feature instead:
+
+[source,xml]
+----
+<featureManager>
+   <feature>persistenceContainer-3.2</feature>
+</featureManager>
+----
+
+For more information, see the link:{url-prefix}/docs/latest/data-persistence-jpa.html[Data persistence with the Jakarta Persistence API] in the Open Liberty docs and the link:https://jakarta.ee/specifications/persistence/3.2/[Jakarta Persistence 3.2 specification document].
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+
+[#run]
+== Try it now
+
+To try out these features, update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 22, Java SE 21, Java SE 17, Java SE 11, and Java SE 8.

```

Comment:
Avoid being misleading here.
```suggestion
To try out these features, update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 22, Java SE 21, Java SE 17, Java SE 11, and Java SE 8.  Java SE 17 or above is required for the Jakarta EE 11 features that are in the beta.

```